### PR TITLE
Run lint in all folders

### LINKED ISSUE #34: chore: update test dependencies to be compatible with python 3.11
Similar to https://github.com/kytos-ng/topology/issues/192

Here's a PR reference https://github.com/kytos-ng/topology/pull/193/files, this `noviflow` NApp doesn't use `pydantic` models so it'll be simpler though.

@Alopalao when I was bumping `2024.1` versions, I just realized this one has been left behind, can you send a PR fixing/upgrading the test dependencies?

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #34 

### Summary

Added all the folders containing python scripts to be inspected by lint

### Local Tests
Run tox

### End-to-End Tests
N/A

### CODE REVIEW

**Comment by Alopalao** [comment]
I added all the folders to be inspected by lint but I did not notice anything that needed to be updated.

File: setup.py [hunk 1]
```diff
@@ -114,8 +115,14 @@ class TestCoverage(Test):
 
     def run(self):
         """Run unittest quietly and display coverage report."""
-        cmd = f"python3 -m pytest --cov=. tests/ {self.get_args()}"
-        call(cmd, shell=True)
+        cmd = "python3 -m pytest --cov=. tests/ --cov-report term-missing"
```
Comment (Alopalao) [inline setup.py:118]: Related to kytos issue [#487](https://github.com/kytos-ng/kytos/issues/487)

**Review by Alopalao — COMMENTED** [review]
I did not notice anything that needed to be updated.
I added all the missing python files to be inspected by the linter.

**Comment by viniarck** [comment]
> I did not notice anything that needed to be updated. I added all the missing python files to be inspected by the linter.

Yes, appreciated you double checking @Alopalao, I turns out I had an older branch that was with py39 for the local dev of this NApp. But, I'm glad you took the opportunity to also fix the adjacent parts here.

**Pull request merged**